app/models/ems_refresh/save_inventory.rb

```diff
@@ -111,8 +111,7 @@ def save_vms_inventory(ems, hashes, target = nil)
 
         found.save!
         h[:id] = found.id
-        found.reload # reload to clear caches and lower memory usage
-        h[:_object] = found
+        h[:_object] = found.class.select(:id, :type).find_by_id(found.id)
```

Comment:
would it make sense to clear out the relations on `found` rather than  reload it?

This current solution always nagged at me.